benchmarks/src/garage_benchmarks/experiments/policies/continuous_mlp_policy.py

```diff
@@ -46,15 +46,17 @@ def continuous_mlp_policy(ctxt, env_id, seed):
     with LocalTFRunner(ctxt, max_cpus=12) as runner:
         env = TfEnv(normalize(gym.make(env_id)))
 
-        action_noise = OUStrategy(env.spec, sigma=hyper_params['sigma'])
-
         policy = ContinuousMLPPolicy(
             env_spec=env.spec,
             name='ContinuousMLPPolicy',
             hidden_sizes=hyper_params['policy_hidden_sizes'],
             hidden_nonlinearity=tf.nn.relu,
             output_nonlinearity=tf.nn.tanh)
 
+        exploration_policy = OUStrategy(env.spec,
```

Comment:
did you miss a refactor here?

Reply:
Yeah, the benchmarks directory got added after I made the initial change. Good catch.